README.md

```diff
@@ -97,12 +97,11 @@ For that you will need to add the keys in the build settings. For more info chec
 
 
 ## MongoDB
-This project is now fully integrated with MongoDB atlas. For it to work, please set the variables in  your .env 
+This project is now fully integrated with MongoDB. For it to work, please set the variables in  your .env , as well as trello_board id and secret-key.
 ```bash
-MONGO_USERNAME=*******
-MONGO_PASS=******
-MONGO_HOST=YOURHOST.mongodb.net
-DEFAULT_DATABASE=db
+MONGO_CONNECTION_STRING=
+TRELLO_BOARD=
```

Comment:
Should this be renamed now? It looks like it's acting as your collection name

Reply:
I need the collection name for the end to end test. I want the user to be able to decide the name of the collection. Maybe we want to have multiple boards?

Reply:
That's a good reason to have an environment variable, I think Jack meant it would make more sense to name it e.g. `MONGO_COLLECTION` since we're not using Trello anymore.